src/DependencyInjection/Configuration.php

```diff
@@ -76,6 +83,20 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('filters')
```

Comment:
Can you please move this to a new method

Reply:
I added method `getFiltersNode`.